packages/mobx/__tests__/v4/base/__snapshots__/array.js.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`very long arrays can be safely passed to nativeArray.concat #2379 1`] = `
+[MockFunction] {
+  "calls": Array [
+    Array [
+      "[mobx] Out of bounds read: 10000",
+    ],
+  ],
+  "results": Array [
+    Object {
```

Comment:
The snapshot include additional information, but it looks like it's the only way to use snapshots on mocks: https://github.com/facebook/jest/issues/2814#issuecomment-347095063